### ISSUE TITLE
DST Staging Review: Figma file doesn't include status eyebrows

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#3765]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** Process list with status
**Staging Review ticket:** [Design System Staging Review: Process list with status](https://github.com/department-of-veterans-affairs/va.gov-team/issues/101657)

## Findings details
**VA.gov Experience Standard - issue:** [User encounters components and patterns that meet best practices.](https://design.va.gov/about/experience-standards/#user-encounters-components-and-patterns-that-meet-best-practices)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**This is an issue with the:** guidance
**High-priority:** yes
**Collab Cycle Reviewer:** @erinrwhite (IA), @allison0034 (Design)

### Description

To reinforce the fact that eyebrows are required elements for the process list, the [Figma file](https://www.figma.com/design/JDFpGLIojfuQwANXScQjqe/VADS-Component-Examples?node-id=35-169&p=f&t=rjwgxrv7owYvmBlr-0) should contain eyebrows above the step headers. These appear on storybook and in the guidance.

<details><summary>Link, screenshot or steps to recreate</summary>

Figma file:
<img width="436" alt="Image" src="https://github.com/user-attachments/assets/b6bd9891-6c88-4840-8afe-ddf05464a1a6" />

[Accessibility guidance:](https://design.va.gov/components/process-list#accessibility-considerations)
<img width="632" alt="Image" src="https://github.com/user-attachments/assets/87b4e957-170c-4b73-99bf-fbbc9866efd7" />

</details>

## Recommended action
Add status eyebrows to the headings in the Figma file.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** [WCAG 1.4.1 Use of Colour - Level A](https://www.w3.org/TR/WCAG21/#use-of-color)
- **Modality:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for DST

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation

**Comment by caw310** [comment]
It seems there is a process list in the Examples file, but no examples with the eyebrow text.

**Comment by LWWright7** [comment]
@babsdenney @danbrady This one is ready for review!

**Comment by LWWright7** [comment]
Got approval from both @danbrady & @babsdenney. Closing this ticket!

![Image](https://github.com/user-attachments/assets/d1e15612-97ef-433d-928c-a4e474066a07)
